libs/stats/odc/stats/_tcw_percentiles.py

```diff
@@ -0,0 +1,95 @@
+"""
+Fractional Cover Percentiles
+"""
+from functools import partial
+from typing import Optional, Tuple, Dict
+import xarray as xr
+import numpy as np
+from odc.stats.model import Task
+from odc.algo.io import load_with_native_transform
+from odc.algo import keep_good_only
+from odc.algo._percentile import xr_percentile
+from odc.algo._masking import _xr_fuse, _fuse_mean_np
+from .model import StatsPluginInterface
+from . import _plugins
+
+
+NODATA = -9999 # output NODATA
+
+
+class StatsTCWPC(StatsPluginInterface):
+    
+    NAME = "ga_tcw_percentiles"
+    SHORT_NAME = NAME
+    VERSION = "0.0.1"
+    PRODUCT_FAMILY = "twc_percentiles"
+
+    def __init__(
+        self,
+        resampling: str = "bilinear",
+        coefficients: Dict[str, float] = {
+            "red": 0.0135, "blue": 0.2021, "green": 0.3102, "nir": 0.1584, "swir1": -0.6806, "swir2": -0.6109
```

Comment:
Sorry my handwriting was so bad and even wrote some  figures wrong... Can you use the line below? Thanks.
```                 
'wetness': {'blue': 0.0315, 'green': 0.2021, 'red': 0.3102, 'nir': 0.1594,
                             'swir1': -0.6806, 'swir2': -0.6109}
```

Reply:
no worries, I think I copied some down wrong too 🤦